### PR TITLE
[12] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by MayowaShoyinka**
## Task: `[12] Style the Contact Us Page using CSS`
- `Changed the icon to an SVG format`
contact-mobile-view-1
contact-mobile-view-2
contact-tablet-view
contact-web-view

### CODE REVIEW

File: submissions/mayowaShoyinka/css/contact.css [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+  background-color:#faf6f2;
+  line-height:1.5;
+  height:100%;  
+}
+/*------------ Header Starts Here-------------*/
+.navigation-header{
+  overflow:hidden;
+  background-color:#116466;
+  padding:25px 56px 0 20px;
+  line-height:50px;
```
Comment (adaugochi) [inline submissions/mayowaShoyinka/css/contact.css:16]: line-height is unitless

File: submissions/mayowaShoyinka/css/contact.css [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+}
+.navigation-header a{
+  float:left;
+  color:#fff;
+  text-align:center;
+  padding:20px;
+  text-decoration:none;
+  font-size:18px; 
+  line-height:25px;
```
Comment (adaugochi) [inline submissions/mayowaShoyinka/css/contact.css:25]: same here

**Review by adaugochi — COMMENTED** [review]
well done

Please add a single space between the property name and the value to make it look readable

File: submissions/mayowaShoyinka/htmlForms/contact-2.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/htmlForms/contact-2.html:1]: You have a contact page already. Why are you creating another? @MayowaShoyinka

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/mayowaShoyinka/htmlForms/contact-2.html (hunk 3 above)
Comment (MayowaShoyinka) [reply submissions/mayowaShoyinka/htmlForms/contact-2.html:1]: the formal contact page was created before thats why i named this contact-2

Comment (ToluAkin) [reply submissions/mayowaShoyinka/htmlForms/contact-2.html:1]: You're not meant to create another one.You should style the one you've created. You will need to delete this newly created file and move the changes to the older contact page created. @MayowaShoyinka

Comment (MayowaShoyinka) [reply submissions/mayowaShoyinka/htmlForms/contact-2.html]: changes made please check @ToluAkin

File: submissions/mayowaShoyinka/htmlForms/contact.html [hunk 4]
```diff
@@ -1,62 +1,77 @@
...
-                <li><a href="#">Contact</a></li>
-
-            </ul>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+<title> Contact || </title>
+    <link rel="stylesheet" href="../css/contact.css">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Serif&family=Raleway:ital,wght@1,300&family=Roboto:ital,wght@1,300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../html/img/font/flaticon.css">
```
Comment (ToluAkin) [inline submissions/mayowaShoyinka/htmlForms/contact.html:8]: Are you using this @MayowaShoyinka ? I don't see this in your PR.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/mayowaShoyinka/htmlForms/contact.html (hunk 4 above)
Comment (MayowaShoyinka) [reply submissions/mayowaShoyinka/htmlForms/contact.html:8]: can i close this PR and upload a new one

Comment (ToluAkin) [reply submissions/mayowaShoyinka/htmlForms/contact.html:8]: Why?

Comment (MayowaShoyinka) [reply submissions/mayowaShoyinka/htmlForms/contact.html:8]: have made changes, but you said you cant see it

the one you talked about: i already closed the PR

Comment (MayowaShoyinka) [reply submissions/mayowaShoyinka/htmlForms/contact.html:8]: please can you check it @ToluAkin

Comment (ToluAkin) [reply submissions/mayowaShoyinka/htmlForms/contact.html:8]: @MayowaShoyinka It is still there.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**